app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,38 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+import { fullTimeDiff } from '../../../lib/utils/dates';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const start = DateTime.fromISO(registrationStart);
+  const [timeLeft, setTimeLeft] = useState(start.diff(DateTime.local(), ['days', 'hours', 'minutes', 'seconds']).toObject());
```

Comment:
Why is this not using the new `fullTimeDiff` util function, now that we have it?

Reply:
good catch